vendor/bat-native-ledger/include/bat/ledger/ledger_client.h

```diff
@@ -25,8 +25,7 @@ using PublisherInfoListCallback =
 using GetNicewareListCallback =
     std::function<void(const Result, const std::string&)>;
 using FetchIconCallback = std::function<void(bool, const std::string&)>;
-using LoadURLCallback = std::function<void(const int, const std::string&,
-    const std::map<std::string, std::string>& headers)>;
+using LoadURLCallback = std::function<void(const ledger::UrlResponse&)>;
```

Comment:
Ditto with regards to `using`, see above

Reply:
this looks good to me

Reply:
Confirmations follows the pattern `using UrlResponse = ledger::UrlResponse` in `confirmations_client.h` to remove `ledger` name space from confirmations code

Reply:
but that is native ledger code not confirmation

Reply:
oh yeah lol